Resources/Prototypes/_Harmony/Entities/Mobs/Aliens/Carps/poisondragon_rift.yml

```
@@ -0,0 +1,24 @@
+﻿- type: entity
+  id: CarpPoisonRift
+  parent: CarpRift
+  name: poison carp rift
+  description: A rift akin to the ones space carp use to travel long distances.
+  components:
```

This file should be namespaced under Prototypes/_Harmony/Entities/Structures/Specific like the parent is

Additionally, suffix
```suggestion
  parent: CarpRift
  name: poison carp rift
  suffix: Harmony
  description: A rift akin to the ones space carp use to travel long distances.
  components:
```